Podcast/ContinueListeningCollectionViewCell.swift

```diff
@@ -0,0 +1,124 @@
+            make.top.equalToSuperview()
+        }
+
+        titleLabel = UILabel()
+        titleLabel.font = ._16SemiboldFont()
+        titleLabel.numberOfLines = 2
+        titleLabel.textColor = .offBlack
+        addSubview(titleLabel)
+        titleLabel.snp.makeConstraints { make in
+            make.leading.equalTo(artworkImageView.snp.trailing).offset(smallPadding)
+            make.trailing.equalToSuperview().inset(padding + smallPadding)
+            make.top.equalTo(artworkImageView.snp.top)
+        }
+
+        descriptionLabel = UILabel()
+        descriptionLabel.font = ._12RegularFont()
+        descriptionLabel.textColor = .charcoalGrey
+        descriptionLabel.numberOfLines = 2
+        addSubview(descriptionLabel)
+        descriptionLabel.snp.makeConstraints { make in
+            make.leading.equalTo(titleLabel.snp.leading)
+            make.trailing.equalTo(titleLabel.snp.trailing)
+            make.top.equalTo(titleLabel.snp.bottom).offset(smallestPadding)
+        }
+
+        timeLeftView = UIView()
+        timeLeftView.backgroundColor = .slateGrey
+        timeLeftView.clipsToBounds = true
+        timeLeftView.layer.cornerRadius = 3
+        addSubview(timeLeftView)
+
+        timeLeftLabel = UILabel()
+        timeLeftLabel.font = ._12SemiboldFont()
+        timeLeftLabel.textAlignment = .center
+        timeLeftLabel.textColor = .offWhite
+        timeLeftView.addSubview(timeLeftLabel)
+
+        timeLeftView.snp.makeConstraints { make in
+            make.bottom.equalTo(artworkImageView.snp.bottom)
+            make.leading.equalTo(titleLabel.snp.leading)
+            make.height.equalTo(timeLeftViewHeight)
+        }
+
+        timeLeftLabel.snp.makeConstraints { make in
+            make.centerY.equalToSuperview()
+            make.leading.trailing.equalToSuperview().inset(smallestPadding)
+        }
+
+        dismissButton = Button()
+        dismissButton.setImage(#imageLiteral(resourceName: "failure_icon"), for: .normal)
+        dismissButton.imageEdgeInsets = UIEdgeInsets.zero
+        dismissButton.addTarget(self, action: #selector(dismissButtonPress), for: .touchUpInside)
+        addSubview(dismissButton)
+
+        dismissButton.snp.makeConstraints { make in
+            make.top.equalToSuperview()
+            make.trailing.equalToSuperview().inset(smallPadding)
+        }
+
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func configure(for episode: Episode) {
+        artworkImageView.setImageAsynchronouslyWithDefaultImage(url: episode.largeArtworkImageURL)
+        titleLabel.text = episode.title
+        descriptionLabel.text = episode.getDateTimeLabelString(includeSeriesTitle: true)
+        if let duration = Double(episode.duration), episode.isDurationWritten {
+            let minsLeft = Int((duration - episode.currentProgress) / 60)
+            timeLeftLabel.text = "\(minsLeft) min left"
```

Comment:
what if there are hours left?

Reply:
![image](https://user-images.githubusercontent.com/20246620/38780782-d6b281bc-40a9-11e8-83ee-f75c9a296306.png)
![image](https://user-images.githubusercontent.com/20246620/38780783-d8f4e316-40a9-11e8-9a81-7185e740ab38.png)
I don't think these are matching up based on the episode current progress